desktop-widgets/qoverload.h

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: (GPL-3.0 OR LGPL-3.0)
+#pragma once
```

Comment:
We tend to use classical include guards. Let's keep things consistent.
(I would have no problem with changing everything to pragma, but also don't see the advantage).

Reply:
I was just lazy.